kv/txn_coord_sender.go

```diff
@@ -35,8 +35,11 @@ import (
 	"github.com/cockroachdb/cockroach/util/hlc"
 	"github.com/cockroachdb/cockroach/util/log"
 	gogoproto "github.com/gogo/protobuf/proto"
+	"github.com/montanaflynn/stats"
```

Comment:
We've got this crazy metrics system in `util/metrics`. Is it not appropriate for this usage?